### PR TITLE
Add Erigon Execution Client

### PR DESCRIPTION
**Pull request by stdevMac**
## Changes:
- Add erigon Execution client to stack

## Types of changes

What types of changes does your code introduce?
- [x] New feature (non-breaking change which adds functionality)

## Testing
- [x] Yes
**In case you checked yes, did you write tests?**

- [ ] Yes
- [x] No

**Comments about testing , should you have some** (optional)
Run smoke tests

### CODE REVIEW

File: templates/services/merge/execution/erigon.tmpl [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    user: root
+    command:
+    - erigon{{with .TTD}}
+    - --override.terminaltotaldifficulty=${TTD}{{end}}
+    - --http
+    - --http.addr=0.0.0.0
+    - --http.port={{.ElApiPort}}
+    - --txpool.disable
+    - ${EC_METRICS_ENABLED:-""}
+    - --chain={{if .SplittedNetwork}}${EL_NETWORK}{{else}}${NETWORK}{{end}}
```
Comment (AntiD2ta) [inline templates/services/merge/execution/erigon.tmpl:33]:
```suggestion
    - --txpool.disable
    - --chain={{if .SplittedNetwork}}${EL_NETWORK}{{else}}${NETWORK}{{end}}
```

File: templates/services/merge/execution/erigon.tmpl [hunk 2]
```diff
@@ -0,0 +1,45 @@
+{{/* erigon.tmpl */}}
+{{ define "execution" }}
+  # Basic erigon's service
+  execution:
```
Comment (AntiD2ta) [inline templates/services/merge/execution/erigon.tmpl:4]:
```suggestion
{{ define "execution" }}
  execution:
```

**Review by AntiD2ta — CHANGES_REQUESTED** [review]
Update /scripts/check-image-updates.sh with Erigon, so we can get the latest release from Erigon as soon as possible.

**Comment by stdevMac** [comment]
@AntiD2ta just solved what you requested, please review it again!

**Review by AntiD2ta — APPROVED** [review]

**Pull request merged**